source/_dashboards/entities.markdown

```diff
@@ -81,7 +81,7 @@ image:
   type: string
 secondary_info:
   required: false
-  description: "Show additional info. Values: `entity-id`, `last-changed`, `last-updated`, `last-triggered` (only for automations and scripts), `position` or `tilt-position` (only for supported covers), `brightness` (only for lights)."
+  description: "Show additional info. Values: `entity-id`, `last-changed` (shows how much time has elapsed from the last change of the entity's value), `last-updated` (shows how much time has elapsed from the last update of the entity's value, even if the value did not change), `last-triggered` (only for automations and scripts), `position` or `tilt-position` (only for supported covers), `brightness` (only for lights)."
```

Comment:
Actually, it turns out the original explanation I was given is probably wrong, and here is the correction:

> This doesn't sound right.
> https://www.home-assistant.io/docs/configuration/state_object
> 
> state.last_updated : Time the state was written to the state machine in UTC time. Note that writing the exact > same state including attributes will not result in this field being updated. 
state.last_changed: Time the state changed in the state machine in UTC time. This is not updated when there are only updated attributes.
> I always think of last_updated as any attribute changed, and last_changed is if the state changed. Exact same values update neither.

https://discord.com/channels/330944238910963714/460846676358332417/1208837969163853834

I can update the PR with correct information if somebody would confirm what is the truth here ;)

Reply:
- `last_changed` is only related to the main entity state.
- `last_updated` is related to the entity as a whole; this includes the state and state attributes. In rare cases, integrations may force an update, even if no values state or attribute states changed.